ext/ddtrace.stub.php

```diff
@@ -0,0 +1,540 @@
+
+    /**
+     * Initialize the tracer and executes the dd_init.php in the sandbox
+     *
+     * @internal
+     * @param string $dir Directory where 'dd_init.php' is located
+     * @return bool 'true' if the initialization was successful, else 'false'
+     */
+    function ddtrace_init(string $dir): bool {}
+
+    /**
+     * Send payload to background sender's buffer
+     *
+     * @internal
+     * @param int $numTraces Trace count. Note that at the moment, the background sender is only capable of sending
+     * exactly one trace
+     * @param array $curlHeaders HTTP Headers
+     * @param string $payload HTTP Body
+     * @return bool 'true' if tracers were successfully sent or if the tracer is disabled, and 'false' if not exactly
+     * one trace was sent or if the procedure was unsuccessful
+     */
+    function dd_trace_send_traces_via_thread(int $numTraces, array $curlHeaders, string $payload): bool {}
+
+    /**
+     * Send raw data to the agent's buffer span
+     *
+     * @internal
+     * @param array $traceArray Trace Array
+     */
+    function dd_trace_buffer_span(array $traceArray): bool {}
+
+    /**
+     * Used to send any already buffered spans to the agent
+     *
+     * @internal
+     * @return int
+     */
+    function dd_trace_coms_trigger_writer_flush(): int {}
+
+    /**
+     * Execute a given internal function
+     *
+     * Internal functions are: init_and_start_writer, ddtrace_coms_next_group_id, ddtrace_coms_buffer_span,
+     * ddtrace_coms_buffer_data, shutdown_writer, set_writer_send_on_flush, test_consumer, test_writers,
+     * test_msgpack_consumer, synchronous_flush, and root_span_add_tag
+     *
+     * @internal
+     * @param string $functionName Internal function name
+     * @param mixed $args,... Arguments of the function
+     * @return bool 'true' if void function was properly executed, else the return value of it
+     */
+    function dd_trace_internal_fn(string $functionName, mixed ...$args): bool {}
+
+    /**
+     * Change the distributed trace id
+     *
+     * @param string|null $traceId New trace id
+     * @return bool 'true' if the change was properly applied, else 'false'
+     */
+    function dd_trace_set_trace_id(?string $traceId = null): bool {}
+
+    /**
+     * Tracks closed spans from user-land and C-level
+     *
+     * @return int Number of closed spans
+     */
+    function dd_trace_closed_spans_count(): int {}
+
+    /**
+     * Check if the tracer's current memory usage is higher than the set limits
+     *
+     * @return bool 'true' if memory is overused, else 'false'
+     */
+    function dd_trace_tracer_is_limited(): bool {}
+
+    /**
+     * Get the compile time of all files compiled up to now (in µs)
+     *
+     * @return int Compile time
+     */
+    function dd_trace_compile_time_microseconds(): int {}
+
+    /**
+     * Get serialized information about closed spans as arrays and send it to the agent (flush). Note that calling
```

Comment:
> send it to the agent (flush).

This function notably does _not_ send anything, but just return the spans (i.e. that makes it a good debugging function)